handlers/discard/README.md

```diff
@@ -0,0 +1,10 @@
+# non-Logger for slog.Logger
+
+The `discard` handler is a placeholder to avoid having to conditionally decide if using a logger
+or not. `discard` will handle Panic() and Fatal() correctly, but everything else will be discarded.
+
+for Panic/Fatal messages, the standard `log.Output()` will be called. fields and callstack are lost.
```

Comment:
it would be nice to have here some wording that the "standard" is stdlib.

Reply:
something like this @karasz ?
```diff
 
-for Panic/Fatal messages, the standard `log.Output()` will be called. fields and callstack are lost.
+for Panic/Fatal messages, the [Go standard logger](https://pkg.go.dev/log) will be called. fields and callstack are lost.
 
 ```